WordPress/src/main/java/org/wordpress/android/localcontentmigration/LocalEligibilityStatusProviderHelper.kt

```diff
@@ -0,0 +1,14 @@
+package org.wordpress.android.localcontentmigration
+
+import org.wordpress.android.fluxc.store.SiteStore
+import org.wordpress.android.localcontentmigration.LocalContentEntityData.EligibilityStatusData
+import javax.inject.Inject
+
+class LocalEligibilityStatusProviderHelper @Inject constructor(
+        private val siteStore: SiteStore,
+): LocalDataProviderHelper {
+    override fun getData(localSiteId: Int?, localEntityId: Int?): LocalContentEntityData {
+        // TODO: check for eligibility on-the-fly
+        return EligibilityStatusData(true, siteStore.sitesCount)
```

Comment:
Would it make sense to unit test the returned object here and also if `siteStore.sitesCount` is called once?

Reply:
One of the benefits of decoupling the provider with these separate helpers is that it makes testing easier. This is especially useful for implementations that have some business logic in them. I'm on the fence about this one in its current state (as it merely forwards a single value), but once the eligibility part is implemented (and not simply hard-coded as `true`), it will certainly have enough complexity to warrant unit tests here.

Reply:
I've pushed some commits adding some tests for the post provider helper, as well as the entity paths and patterns.